BM/instruction-check/feature_list.py

```diff
@@ -145,6 +145,14 @@
         "cpuid": ['7', '0', '1', '0', 'a', '7'],
         "platforms": {"SRF", "CWF"}
     },
+    "lam": {
+        "cpuid": ['7', '0', '1', '0', 'a', '26'],
+        "platforms": {"SRF", "CWF"}
+    },
+    "lass": {
+        "cpuid": ['7', '0', '1', '0', 'a', '6'],
```

Comment:
Can you sort it by number? For example: ['7', '0', '1', '0', 'a', '6'] is on above of ['7', '0', '1', '0', 'a', '7'].

Reply:
updated with sorting by CPUID number.